app/javascript/pages/fullArticle.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { Text } from '@patternfly/react-core';
+import colors from '../utils/colors';
+import previewImage from '../../assets/images/img1.png';
+const styles = {
+  title: {
+    textAlign: 'center',
+    alignSelf: 'center',
+    color: colors.appBlue,
+    fontSize: '50px',
+    fontWeight: 'bolder',
+    marginInline: '10%'
+  },
+  img: {
+    marginInline: '25%',
+    marginTop: '2%',
+    marginBottom: '2%',
+    borderRadius: 10
+  },
+  body: {
+    textAlign: 'center',
+    alignSelf: 'center',
+    color: colors.appGrey,
+    fontSize: '30px',
+    marginInline: '20%'
+  }
+};
+const FullArticle = () => {
+  const { state } = useLocation();
+  return (
+    <>
+      <Text style={styles.title}>{state.article.title}</Text>
```

Comment:
nit: can use destructuring here:

```suggestion
  const {
    state: {
      article: { title, body }
    }
  } = useLocation();
  return (
    <>
      <Text style={styles.title}>{title}</Text>
```

btw, what are we going to do with the full page reload? no data will be available from the router state. can you open an issue for that and we may address it in a different PR

Reply:
added the destructuring and yes we shall use a separate call for it, I will open a issue for the same